src/modules/universal-discovery/components/content-table/content.table.header.component.js

```diff
@@ -6,6 +6,7 @@ const ContentTableHeaderComponent = () => {
 
     return (
         <div className="c-content-table-header__list-headers">
+            <div className="c-content-table-header__list-header c-content-table-header__list-header--icon" />
```

Comment:
Empty div?

Reply:
Yes, it's table-like structure, so we want to have header for all columns and icons have separate column. However, we don't want to have any label in header for icons' column.